iceberg/iceberg-common/src/main/java/org/apache/gravitino/iceberg/common/ops/IcebergTableOps.java

```diff
@@ -66,6 +68,7 @@ public class IcebergTableOps implements AutoCloseable {
   private final IcebergCatalogBackend catalogBackend;
   private String catalogUri = null;
   private Map<String, String> catalogConfigToClients;
+  private Map<String, String> hadoopConfigMap;
```

Comment:
the variable name is misleading, better to use  something like `icebergCatalogProperties`?